WorldBank-Transport/DRIVER
# Fix loss of polygon filtering caused by Created Date filter

## Overview
When adding the Created Date filter, the implementation inadvertently removed functionality for filtering by polygon. Additionally, the implementation renamed the "Date Range" filter to "Occurrence Date", rather than the intended "Occurred Date" (https://github.com/WorldBank-Transport/DRIVER/pull/701#issuecomment-417433929)

### Checklist


### Demo
![screen shot 2018-09-07 at 3 12 59 pm](https://user-images.githubusercontent.com/1032849/45238681-b4ac9e80-b2b0-11e8-8235-3cbc4b53fd44.png)

## Testing Instructions
- In the map view, with some records visible, select the polygon or rectangle tool and draw a shape that excludes some records
  - Records not encompassed within the drawn shape should be hidden
